### DISCUSSION TITLE
Table Tents

### DISCUSSION THREAD

**Discussion by lostcarpark** [LVerhulst4321/PlanZ#57] [Ideas]
For Eastercon we used the table tents feature.

The feature seems to be set up for the way one particular convention does table tents, so we made some modifications to suit Eastercon.

The default "fold in two" tents didn't suit so we tweaked the CSS for fold in 3 tents.

We also dropped the separator pages, as they seemed to waste a lot of paper.

Finally, I fixed a bug that caused the date to not print for either the item or the participant's next item.

I haven't done anything about it yet, but the pages print correctly on Firefox, but don't seem to line up with pages in Chrome or Safari.

I have a list of changes I'd like to make to table tents.

- Offer choice of fold in 2 or fold in 3. Update folding instructions to match selected type.
- Using `@media` CSS styles to make it work correctly on all browsers.
- Add options to select whether to print tents for the whole convention, a single day, or a single session.
- Option print tents for a single room (combined with above filters).
- Option to print all table tents for one participant.
- Option to print one table tent per participant, with their schedule on the back, which they would be expected to bring to all their sessions (I think this has been proven not to work, but some conventions may want to persist with it).
- As we move towards more hybrid participation, offer options to exclude virtual participants (and virtual panels), which don't need table tents (not sure if we currently track virtual participants).
- Someone asked could the back of the table tent indicate whether the session would be streamed or not (we were using a tag to indicate that, but there is probably a better way).

**Comment by bcholmes** [comment]
It's great to see live experience feed back into the implementation. A coupl'a thoughts:

1. I was thinking that a good way to print a table tent for one particular session might be to start that process on the Create/Edit session page. Have an extra toolbar button that's basically "Print table tent". That makes the "select the session you want" interface easier.
2. Your point about online/hybrid is really good: we probably need to have a consistent way of marking panels as online / in-person / hybrid, as well as marking participant assignments as online / in-person.
